docs/user/interactive_stopping_and_saving.rst

```diff
@@ -0,0 +1,118 @@
+.. _stopping_and_saving:
+
+Stopping and Saving
+===================
+
+If you know you're not going to be actively working on your project, it's good
+manners to :ref:`save your work<saving>` and stop the interactive environment, so that
+you can release the resources you were consuming. You can do this from the Environments
+tab on the RenkuLab UI.
+
+Sometimes an environment will get stuck pending launch (e.g. because you requested
+GPUs and they are not yet available). In this case you can view the status and
+logs to see if there's a useful message. In the case of pending GPUs, you can
+safely wait for the resources to become available. However in other cases, more
+commonly when you are trying to customize your environment, you might notice an
+issue via the logs, and want to stop the launch so that you can fix the problem
+and try again. However, for these stuck notebooks it is not yet possible.
+You can reach out to us on `discourse <https://renku.discourse.group>`_ in this
+case.
+
+.. _saving:
+
+Saving your work
+----------------
+
+Interactive environments are kept running as long as they in use. In the default RenkuLab configuration, idle environments are stopped after 24 hours of inactivity. Furthermore, interactive environments can crash, for example if you run a process that eats more memory than you've allocated. Thus, it's best to save often.
+down without warning. They also might crash if you run a process that eats more
+memory than you've allocated. Thus, it's best to save often.
```

Comment:
```suggestion
```
Looks like his line was added by accident.

Reply:
oops - I think I accidentally introduced this with my github commit suggestion